### DISCUSSION TITLE
Fingerprint changes based on external monitor?

### DISCUSSION THREAD

**Discussion by g-h-miles** [thumbmarkjs/thumbmarkjs#32] [General]
First, thank you so much to all contributors for this library. Is it intentional for the fingerprint to change if the browser is moved to an external monitor? If so, what is the use case for that? Thanks, again.

**Comment by ilkkapeltola** [comment]
That's not intentional.

I'm pretty sure there's a media query that I didn't realize is affected by screen capabilities or something.
Could you get the JSON from https://www.thumbmarkjs.com/ on both monitors and pass them to https://jsonformatter.org/json-compare for comparison, and let me know where the JSON object differs. I'll look into it.

**Comment by g-h-miles** [comment]
It does not occur on my Lenovo + Dell monitor setup but does on my Macbook + Dell monitor setup. I was able to prevent the change by commenting out the 'colorDepth' (line 9) and 'color' (line 30) attributes in the screen component. 

When I get back to my Macbook this afternoon, I will send over the JSON difference. I will note that Fingerprintjs also overlooks this.

**Comment by g-h-miles** [comment]
![Screenshot 2024-03-14 at 9 07 48 PM](https://github.com/thumbmarkjs/thumbmarkjs/assets/149526550/a00b6ff8-279f-4c2f-bc42-100d1f19ba2a)

**Reply by ilkkapeltola** [reply]
looks like it's the color media query. I'll remove that.

**Comment by ilkkapeltola** [comment]
Removed the color media query in 0.14.5
